### PR TITLE
mywebdevelopment

### PR DESCRIPTION
**Pull request by michaelyohannes123**
I accidentally pushed my comment functionality code to master so what I did was place an "edited here" comment specifying the files where I did a significant amount of editing to in making the functionality. Sorry for the inconvenience this may have caused!

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/ListDataServlet.java [hunk 1]
```diff
@@ -27,7 +27,7 @@
 import com.google.appengine.api.datastore.Query;
 import com.google.appengine.api.datastore.Query.SortDirection;
 import com.google.sps.data.Comment;
-
+//edited here
```
Comment (mk-valentine) [inline portfolio/src/main/java/com/google/sps/servlets/ListDataServlet.java:30]: why comments_count = 20 if there are no comments?

File: portfolio/src/main/webapp/comments.html [hunk 2]
```diff
@@ -7,6 +7,7 @@
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <script src="comments_script.js"></script>
   </head>
+  <!-- get a max of 20 comments initially, edited here -->
```
Comment (mk-valentine) [inline portfolio/src/main/webapp/comments.html:10]: ahh I see, maybe also add this comment to the 
if(co == null){
      comments_count = 20;
    }
logic in the java code?

**Review by mk-valentine — APPROVED** [review]
No worries, thanks for explaining! Looks good. Just remove the "edited here" comments before merging.

**Review by foodhype — APPROVED** [review]
No worries! Thanks for explaining.

**Pull request merged**